tutorials/task_workflows/item_status/chapter.md

```diff
@@ -28,21 +31,38 @@ dataset.items.update_status(status=dl.ItemStatus.DISCARDED, filters=filters)
 item_ids = ['id1', 'id2', 'id3']
 dataset.items.update_status(status=dl.ItemStatus.COMPLETED, item_ids=item_ids)
 ```
+#### 2. use task entity (this way recommended for items in the same task)  
+
+```python
+# With filter
+filters = dl.Filters(field='annotated', values=True)
+item_ids = [item.id for item in dataset.items.list(filters=filters).all()]
+# With list of item ids
+item_ids = ['id1', 'id2', 'id3']
+task.set_status(status=dl.ItemStatus.APPROVED, operation='create', items=item_ids)
+```
 ### 3. Clear status from an item (no-status)  
 Clearing a status from an item will make it available again for work in the respective task, and the worker (annotator)  
 will see the item in the assignment (either QA or annotations).  
 
 ```python
 # Clear status for completed/approved/discarded
-item.update_status(dl.ITEM_STATUS_COMPLETED, clear=True)
+item.update_status(dl.ITEM_STATUS_DISCARDED, task_id=task.id, clear=True)
+item.update_status(dl.ITEM_STATUS_APPROVED, assignment_id=assignment.id, clear=True)
+item.update_status(dl.ITEM_STATUS_COMPLETED, clear=True)  # this will work if the item is included in only one task
 ```
-### 4. Manage task statuses  
-The statuses in a task can be managed, add new custom statuses and remove existing ones.  
-  
-  
-<div style="background-color: lightblue; color: black; width: 50%; padding: 10px; border-radius: 15px 5px 5px 5px;"><b>Note</b><br>  
-Changing the statuses in the task doesn't change the status on any items that already received a status.  
+### 4. Add task statuses  
 
 ```python
-...
+# Create annotation task with new statue
+task = dl.tasks.create(
+    task_name='<task_name>',
+    due_date=datetime.datetime(day=1, month=1, year=2029).timestamp(),
+    assignee_ids=['<annotator1@dataloop.ai>', '<annotator2@dataloop.ai>'],
```

Comment:
Why the <>? this can be confusing. It should be either 'annotator1@dataloop.ai' or '<email>'